File: NetCore/AspNetCore/Data/Entities/Models/Classes.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace AspNetCore.Data.Entities.Models
{
    public partial class Classes
    {
        public Classes()
        {
            Enrollments = new HashSet<Enrollments>();
        }

        public int ClassId { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }

		[JsonIgnore]
        public virtual ICollection<Enrollments> Enrollments { get; set; }
    }
}

File: go-mux/apps/api/models/classes.go
package models

//Classes is for class table on DB
type Classes struct {
	ClassesID   int32  `gorm:"primary_key;auto_increment" json:"id"`
	Title       string `gorm:"size:64;not_null" json:"title"`
	Description string `gorm:"type:text;not_null" json:"description"`
}
